### DISCUSSION TITLE
Search page in next.js ssg

### DISCUSSION THREAD

**Discussion by MejanH** [vercel/next.js#14742] [Help]
# How can I add search bar in next SSG site

Should I do with the react way? or is there any plugin or something for adding search bar?

**Comment by leerob** [comment]
Check out [Algolia](https://www.algolia.com/)! There are quite a few resources out there about integrating Algolia with React.

**Reply by IRediTOTO** [reply]
Yes, this is BEST solution. Create your own will be lost time :D Algolia is super fast

**Reply by MejanH** [reply]
> Check out [Algolia](https://www.algolia.com/)! There are quite a few resources out there about integrating Algolia with React.

Thanks for suggesting Algolia, learned some new. I thought Nextjs would create a plugin like gatsby if they don't have one. 

is there going be any plugin or function like getStaticProps in the future for searching contents? or should I depend on third-party libraries like Algolia forever?
